README.md

```diff
@@ -1 +1,107 @@
-# hybrid-android-example-app
+# Klarna In-App SDK - Hybrid Android Demo App
+![Platforms](https://img.shields.io/badge/platform-android-lightgrey)
+[![GitHub](https://img.shields.io/github/license/klarna/klarna-mobile-sdk)](https://github.com/klarna/klarna-mobile-sdk/blob/master/LICENSE)
+
+This is an example Android app to demonstrate the hybrid integration of [Klarna In-App SDK](https://github.com/klarna/klarna-mobile-sdk).
+
+The Klarna In-App SDK allows merchants to natively integrate Klarna’s products into their apps. This means that you can add services like Klarna Payments to your app and let your customers pay with our payment methods.
+
+**Looking for Klarna Checkout Android Demo App?** Check out the [Klarna Checkout Android Demo App](https://github.com/klarna/kco-android-example-app) repo instead.
+
+## What Does Klarna In-App SDK Offer?
+Klarna In-App SDK offers two integration approaches: **Native integration** and **Hybrid integration**.
```

Comment:
add after **Hybrid integration** "(showcased in this demo app)" or something like that.

Reply:
done